tests/ops/fixtures/application_fixtures.py

```diff
@@ -592,6 +590,291 @@ def erasure_policy_two_rules(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.device.ip_address").value,
+            "rule_id": user__device__ip_address_rule.id,
+        },
+    )
+    user__gender_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__gender Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Masked"},
+            },
+        },
+    )
+    user__gender_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.gender").value,
+            "rule_id": user__gender_rule.id,
+        },
+    )
+
+    user__contact__address__city_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__address__city Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Masked"},
+            },
+        },
+    )
+    user__contact__address__city_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.address.city").value,
+            "rule_id": user__contact__address__city_rule.id,
+        },
+    )
+    user__contact__address__country_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__address__country Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Mars"},
+            },
+        },
+    )
+    user__contact__address__country_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.address.country").value,
+            "rule_id": user__contact__address__country_rule.id,
+        },
+    )
+    user__contact__email_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__email Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": f"{uuid4().hex}_MASKED@email.com"},
```

Comment:
We have a way to support the format preservation, try something like this instead
```
"masking_strategy": {
            "strategy": RandomStringRewrite.name,
            "configuration": {
                "length": 20,
                "format_preservation": {
                    "suffix": "@email.com"
                }
            }
        }
```